runtime/integration-tests/src/generic/env.rs

```diff
@@ -60,6 +60,9 @@ impl<Event> Blocks<Event> {
 
 /// Define an environment behavior
 pub trait Env<T: Runtime> {
```

Comment:
Could we also add just: 
```suggestion
pub trait Env<T: Runtime>: Default {
```

To enable the possibility of creating a default without knowing the env type.

Reply:
Will add this as soon as I figure out the issue with benchmarks. Thank you!

Reply:
Added this and a couple of more changes to the `Env` trait. PTAL